packages/create-react-app/index.js

```diff
@@ -105,7 +105,7 @@ function createApp(name, verbose, version) {
   checkAppName(appName);
 
   if (!pathExists.sync(name)) {
-    fs.mkdirSync(root);
```

Comment:
So, does that make sence to add fs-extra. 

Even i think it would be better since there'll be more use cases where it's helpful.